markdown_validator/.env.example

```diff
@@ -0,0 +1,3 @@
+MODEL='oh-2.5m7b-q51'
+OPENAI_API_BASE_URL="http://localhost:8000/v1" # LLM Studio
```

Comment:
I see you set these but you don't necessarily use them, not a blocker but a clean up opportunity if you wanna follow up with that